http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -415,6 +427,81 @@ object ClusterTemplateValues {
         .getOrElse(""),
       clusterResourceConstraints.map(_.memoryLimit.toString).getOrElse("")
     )
+
+  def fromCreateCluster(createCluster: CreateCluster,
```

Comment:
I think this method name is a bit misleading, It leads me to believe it is the equvialent of an `unapply` for `CreateCluster`, where as it is really wrapping it into another datatype

Reply:
what would you recommend as a better naming? I think over time, these methods should go away as we slowly deprecating generic `Cluster` being used everywhere